tests/test_database.py

```diff
@@ -87,15 +88,18 @@ def test_add_workout(self):
         db_workout = database.get_workout_class_by_name(new_workout.name)
 
         # Setting our current workout object's id as mongodb id of user in string format
-        new_workout.creator = str(db_user.id)
-        # Sets the creator's id
-        database.set_workout_creator(new_workout.id, new_workout.creator)
-        new_workout.creator = str(db_user.id)
-        self.assertTrue(str(db_user.id) == new_workout.creator)
+        new_workout.creator_id = str(db_user.id)
+
+        # Sets the id
+        new_workout.id = db_workout.id
+
+        #sets creator's id
+        new_workout.creator_id = str(db_user.id)
+        self.assertTrue(str(db_user.id) == new_workout.creator_id)
```

Comment:
Don't need str() since id gets converted within the `get_user_class_by_username` class.